tests/integration/proxy/grpcproxy/cluster_test.go

```diff
@@ -67,6 +71,47 @@ func TestClusterProxyMemberList(t *testing.T) {
 	if mresp.Members[0].ClientURLs[0] != cts.caddr {
 		t.Fatalf("mresp.Members[0].ClientURLs[0] expected %q, got %q", cts.caddr, mresp.Members[0].ClientURLs[0])
 	}
+
+	//test proxy member up/down
+	newMemberAddr := "127.0.0.2:6789"
+	grpcproxy.Register(lg, cts.c, prefix, newMemberAddr, 7)
+	// wait some time for proxy update members
+	time.Sleep(time.Second)
```

Comment:
nit: I think we can switch to `time.Sleep(200 * time.Millisecond)` for all `waits` in this test. 1s seams too long.